docs/JSON-RPC.md

```diff
@@ -0,0 +1,435 @@
+
+# Jamulus JSON-RPC Documentation
+
+<!--
+This file is automatically generated from the source code.
+Do not edit this file manually.
+See `tools/generate_json_rpc_docs.py` for details.
+-->
+
+A JSON-RPC interface is available for both Jamulus client and server to allow programmatic access.
+To use the JSON-RPC interface, run Jamulus with the `--jsonrpcport <port> --jsonrpcsecretfile /file/with/a/secret.txt` options.
```

Comment:
Generation of this secret should be linked.

Reply:
Do you mean describing how to generate the secret? I've added that now.

Reply:
Yes. Thanks.